commenter.js

```diff
@@ -15,19 +15,19 @@ var config = require('./config');
  * https://developer.github.com/v3/activity/events/types/#pullrequestevent
  */
 function postPullRequestComment(commentURL, comment, filename, commitSHA, line, done) {
-    sendRequest({
+
+    request.post({
         url: commentURL,
-        method: 'POST',
         headers: {
             'User-Agent': config.brand,
             'Authorization': 'token ' + config.token
         },
-        body: JSON.stringify({
+        form: {
```

Comment:
Actually it's not related to this pull request at all. I'll ask in the discussion area anyway since it's on my mind, but we shouldn't block on it.